prow/crier/reporters/gerrit/reporter.go

```diff
@@ -46,11 +47,16 @@ const (
 	hourglass  = "⏳"
 	prohibited = "🚫"
 
-	defaultProwHeader         = "Prow Status:"
-	jobReportFormat           = "%s %s %s - %s\n"
-	jobReportFormatWithoutURL = "%s %s %s\n"
-	errorLinePrefix           = "NOTE FROM PROW"
-	jobReportHeader           = "%s %d out of %d pjs passed! 👉 Comment `/retest` to rerun only failed tests (if any), or `/test all` to rerun all tests\n"
+	defaultProwHeader               = "Prow Status:"
+	jobReportFormat                 = "%s [%s](%s) %s\n"
+	jobReportFormatUrlNotFound      = "%s %s (%s) %s\n"
+	jobReportFormatWithoutURL       = "%s %s %s\n"
+	jobReportFormatLegacyRegex      = `^(\S+) (\S+) (\S+) - (\S+)$`
+	jobReportFormatRegex            = `^(\S+) \[(\S+)\]\((\S+)\) (\S+)$`
+	jobReportFormatUrlNotFoundRegex = `^(\S+) (\S+) \(\S+\) (\S+)$`
+	jobReportFormatWithoutURLRegex  = `^(\S+) (\S+) (\S+)$`
```

Comment:
there are pretty strong contract between these pairs, would it make sense to have unit tests enforcing them?

Reply:
Done